src/components/Modal/README.md

```diff
@@ -42,7 +42,11 @@ Modals are overlays that prevent merchants from interacting with the rest of the
 
 Passing an API key to the [app provider component](https://polaris.shopify.com/components/structure/app-provider#section-initializing-the-shopify-app-bridge) causes the modal component to delegate to the [Shopify App Bridge](https://help.shopify.com/en/api/embedded-apps/app-bridge) instead of rendering as it would in a stand-alone application.
 
-Note in the props table that a number of properties are only available in stand-alone applications, and won't work in an embedded context.
+In an embedded application context, not all documented properties are available. Some properties are only available in stand-alone applications.
```

Comment:
I think that we can cut the content on lines 45 - 47 down, to reduce repetition and keep things concise: 

```
Some properties aren’t available in an embedded application context. These props, such as `children`, include “(stand-alone app use only)” in their description.
``` 

I'm not up-to-speed on all the context here, but I'm wondering if we've considered adding a pill for this like we have for props available only for embedded apps:

![alt](https://screenshot.click/21-39-aqv51-9ildd.jpg)

This would add more visibility and consistency.

Reply:
This would be really nice. Probably outside of the scope of this PR.